clang/include/clang/Lex/Preprocessor.h

```diff
@@ -767,7 +762,7 @@ class Preprocessor {
   /// \#included, and macros currently being expanded from, not counting
   /// CurLexer/CurTokenLexer.
   struct IncludeStackInfo {
-    enum CurLexerKind           CurLexerKind;
+    LexerCallback CurLexerCallback;
```

Comment:
Can you align this with the other nearby declarations?

Reply:
That would break clang-format :-) If we want to keep the original alignement, I need to add clang-format comments, should I do it that way?

Reply:
Go ahead and break clang-format; it's not paying attention to the surrounding coding style which is what we're supposed to do per community policy. No need to add clutter comments.